### PR TITLE
dark mode feature

### PR DESCRIPTION
**Pull request by mbogdanel**
Ended up opening a second PR, coudn't reverse the format changes made by Prettier. Need to pick up some git skills.
-added defer to script tag
-simplified changes on infima.css (disabled Prettier on vscode)

darkmode

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -25,6 +25,14 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Breads</title>
+    <!-- Below JSDelivr CDN and Widget script for DarkMode -->
+    <script src="https://cdn.jsdelivr.net/npm/darkmode-js@1.5.7/lib/darkmode-js.min.js" defer></script>
+    <script>
```
Comment (aTmb405) [inline public/index.html:30]: Please add `defer` to this script tag too

**Review by aTmb405 — CHANGES_REQUESTED** [review]
Looks great! Thanks for doing this. I just added one comment. A second `defer` needs to be added to another `script` tag.

File: public/index.html (hunk 1 above)
Comment (mbogdanel) [reply public/index.html]: Done. 
Thank you and all the gang in the back for the opportunity of doing this.
Gonna try to pick up the next one, though I'm new and everything looks confusing.

**Review by aTmb405 — APPROVED** [review]
Looks great! Thanks for the changes

**Pull request merged**